### ISSUE TITLE
[Bug Report] Aria-Expanded on V-Expansion-Panel-header not being caught by screen readers

### ISSUE THREAD

**Issue by flpehr0** [vuetifyjs/vuetify#13354]
### Environment
**Vuetify Version:** 2.4.6
**Vue Version:** 2.6.11
**Browsers:** Google Chrome
**OS:** Windows 10

### Steps to reproduce
- Turn on a screen reader (NVDA/JAWS)
- Activate an expansion panel header both by tabbing/enter and clicking to open and close the panel 

### Expected Behavior
- When the expansion panel is expanded or closed the screen reader should read "expanded"/"collapsed". 

### Actual Behavior
- The screen reader does not announce when a panel has been expanded or collapsed. 

### Reproduction Link
<a href="https://codepen.io/flpehr0/pen/poRREOV?editors=101" target="_blank">https://codepen.io/flpehr0/pen/poRREOV?editors=101</a>

### Other comments
In line 4 of VExpansionPanel.spc.ts.snap you can see that the aria-expanded value is applied on the div with "v-expansion-panel" when it should be on the button element "v-expansion-header" immediately below it. https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/components/VExpansionPanel/__tests__/__snapshots__/VExpansionPanel.spec.ts.snap

For more information on aria-expanded and accessible accordions: 
https://www.w3.org/TR/wai-aria-practices/#accordion 
https://www.accessibility-developer-guide.com/examples/sensible-aria-usage/expanded/
